magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -223,9 +229,21 @@ def run_with_flags(generator):
   # files.
   date_and_time = time.strftime('%Y-%m-%d_%H%M%S')
   digits = len(str(FLAGS.num_outputs))
+
   for i in range(FLAGS.num_outputs):
     generated_sequence = generator.generate(input_sequence, generator_options)
 
+    if FLAGS.exclude_primer_midi:
+      # Delete any notes with start_time less than start_time of generated
+      # sequence. Otherwise, subtract length of removed input_sequence from
+      # start_time and end_time of remaining notes.
+      for index, note in enumerate(generated_sequence.notes[:]):
+        if note.start_time < start_time:
+          del generated_sequence.notes[index]
```

Comment:
Thanks @cghawthorne. Ok, so maybe `remove` would be the right approach here even if it's more costly. I'll take a look at sequences_lib.extract_subsequence. Appreciate it!